nvflare/fuel/f3/stats_pool.py

```diff
@@ -95,9 +97,15 @@ def from_dict(d: dict):
         b.count = d.get(_KEY_COUNT, 0)
         b.total = d.get(_KEY_TOTAL, 0)
         m = d.get(_KEY_MIN)
-        b.min = m if m else None
+        if isinstance(m, str):
+            b.min = None
+        else:
+            b.min = m
```

Comment:
If "m" is a string, then we set b.min = None?
If not a string, then b.min set to m.

Question here, what type do we support for "m", any type?
If min = "5" (a string) then we just set b.min = None?
Or should we raise Exception in that case?

Reply:
No, if you look at the code of to_dict, the only possible types are str (representing None) or float.

Reply:
got it, then we are assuming, the dict pass in here is a dict generated from "to_dict" method.